ci-operator/config/openstack-k8s-operators/infra-operator/openstack-k8s-operators-infra-operator-master.yaml

```diff
@@ -0,0 +1,47 @@
+base_images:
+  golangci-lint:
+    name: golangci-lint
+    namespace: ci
+    tag: v1.46.0
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
```

Comment:
Lets add pre-commit job here, since infra-operator already have a .pre-commit-config.yaml merged[1]?

1. Adding the job [2]
2. adding the image needed for pre-commit job [3]
3. Adding reference for build_root from repo [4] 


[1] https://github.com/openstack-k8s-operators/infra-operator/blob/master/.pre-commit-config.yaml
[2] https://github.com/openshift/release/blob/ea90195a5b6e21bd2c3e8edd32c59b49b3d15ea6/ci-operator/config/openstack-k8s-operators/manila-operator/openstack-k8s-operators-manila-operator-main.yaml#L64-L70
[3] https://github.com/openshift/release/blob/ea90195a5b6e21bd2c3e8edd32c59b49b3d15ea6/ci-operator/config/openstack-k8s-operators/manila-operator/openstack-k8s-operators-manila-operator-main.yaml#L29-L33
[4] https://github.com/openshift/release/blob/ea90195a5b6e21bd2c3e8edd32c59b49b3d15ea6/ci-operator/config/openstack-k8s-operators/manila-operator/openstack-k8s-operators-manila-operator-main.yaml#L10-L11

Reply:
thanks for adding the info here. I had the same question for the openstack-baremetal-operator pull request i'm working on. the repo already has the pre commit config merged

Reply:
ack, I added pre commit check to the tests and removed the ones which are already included in it.

Reply:
(mark this conversation resolved? you have addressed the comment)